index.html

```diff
@@ -71,5 +71,87 @@ <h2>Software Developer</h2>
 
```

Comment:
Congrats on the hard work you put into that project ^^!
Your PR looks so professional with a descriptive message. 💯 😄 

_Area to improve:_
_- Add more one-line descriptive commits._